src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -319,17 +319,13 @@ public static bool ContainsAnonymousType([NotNullWhen(returnValue: true)] this I
 
         private static bool ContainsAnonymousType(INamedTypeSymbol type)
```

Comment:
Can we remove this method and overload on line 309?

Reply:
Kindly ping on this question

Reply:
i don't really know what you mean.  How would we remove it?

Reply:
I'm suggesting we delete lines 309 through 332.

Reply:
Why would we do that?

Reply:
Arg. My bad. When I'd done a FindAllRefs, I'd only found those two methods referencing each other... But in actuality they are used...
Good to go then. Thanks

Reply:
basically, i don't understand the motivation here.  why would we remove these helpers?  and what is your suggestion on what to do about the breakage?

Reply:
got it :)  ok.